examples/glue/glue_example.yaml

```diff
@@ -33,7 +33,7 @@ pretrain_hparams:
         label: bert_pre_training
         eval_dataset:
           streaming_c4:
-            remote: s3://allenai-c4/mds/1/
+            remote: my-s3-bucket-name
```

Comment:
Since this change seems a bit out of scope, I'm wondering what the motivation is? Also are we sure we don't want to point to the public C4 bucket in our basic GLUE example?

At the very least I might add a comment here flagging that the user needs to set up the remote data source and replace the placeholder.

Reply:
Whoops this is a mistake guess I had some carried over changes from another branch. Nice catch